submissions/xT0PERx/html-movie-seat-booking/main.css

```
@@ -0,0 +1,357 @@
+html {
+  display: grid;
+  justify-content: center;
```

Too many elements with dispensable `display: grid`.
Avoid adding styles to the tags <html> or <body>
If you need to center content on the page:
```suggestion
<body>
    <div class="container">
    </div>
</body>

.container {
    margin: o auto;
}
```